src/docs/development/tools/sdk/releases.md

```diff
@@ -15,6 +15,13 @@ toc: false
 The {{site.sdk.channel | capitalize }} channel contains the
 most stable Flutter builds. See [Flutter’s channels][] for details.
 
+{{site.alert.note}}
+  As of release 1.17 (stable), we have updated our release process.
+  This includes how we name the releases and their corresponding files.
```

Comment:
```suggestion
  including changes to the names of releases and their download files.
```

Reply:
"how we name" makes more sense in this context to me.  if we want to be more specific we can say "we've changed our versioning schema," but I think "how we name" makes sense.

Reply:
Maybe:

...changes to how we name releases and their download files.

Reply:
I was trying to avoid too much "we" and unadorned "This". Although given the name of the link I didn't avoid "we" entirely.